gem/lib/earlgrey/version.rb

```diff
@@ -14,5 +14,5 @@
 #  limitations under the License.
 
 module EarlGrey
-  VERSION = '0.2.1'.freeze unless defined? ::EarlGrey::VERSION
+  VERSION = '0.2.2'.freeze unless defined? ::EarlGrey::VERSION
```

Comment:
Hah, sorry I forgot what was the decision to be made, is it to be sync'd or not?

Note that we are doing more mainstream releases than gem releases.